web-transport-quinn/Cargo.toml

```diff
@@ -15,7 +15,7 @@ categories = ["network-programming", "web-programming"]
 default = ["aws-lc-rs"]
 aws-lc-rs = ["dep:aws-lc-rs", "quinn/aws-lc-rs", "rustls/aws-lc-rs"]
 ring = ["dep:ring", "quinn/ring", "rustls/ring"]
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+tls12 = ["rustls/tls12"]
```

Comment:
What's the goal? QUIC doesn't support TLS 1.2

Reply:
It doesn't matter right? Features are meant to be additive, such that my library doesn't care what other features your application has enabled.